"""Error handling for list_subtraction"""


class listSubError(Exception):
    pass